terraform/database/variables.tf

```diff
@@ -0,0 +1,20 @@
+variable "project" {
+  type        = string
+  description = "The Google Cloud Platform project name"
+}
+
+variable "region" {
+  default = "us-central1"
+  type    = string
```

Comment:
Usually "type" is the first entry, then "description", then "default" if any

Reply:
Huh, TIL. Thanks!